scripts/consul_client_bootstrap.sh

```diff
@@ -108,98 +119,70 @@ fi
-CONSUL_UPSTART_CONF="${S3SCRIPT_PATH}/consul-client.conf"
-CONSUL_UPSTART_FILE="/etc/init/consul.conf"
-
-#CONSUL VARIABLES
-echo  "Bootstrapping ${PROGRAM}"
-EX_CODE=$?
-
-
-
-## Install dependencies
-apt-get -y install curl unzip jq
-chkstatus
-
-echo "Fetching Consul... from $CONSULDOWNLOAD"
+CONSUL_SERVICE_CONF="${S3SCRIPT_PATH}/consul.service"
+CONSUL_SERVICE_FILE="/etc/systemd/system/consul.service"
 
-curl -L $CONSULDOWNLOAD > /tmp/consul.zip
+echo "Updating package list..."
+apt-get -y update
 chkstatus
 
-echo "Unpacking Consul to: ${BINDIR}"
-unzip  /tmp/consul.zip -d  /usr/local/bin
-chmod 0755 /usr/local/bin/consul
-chown root:root /usr/local/bin/consul
+echo "Installing dependencies..."
+apt-get -y install curl unzip
 chkstatus
 
 echo "Creating Consul Directories"
 mkdir -p $CONSULCONFIGDIR
 mkdir -p $CONSULDIR
 mkdir -p $CONFIGDIR
 mkdir -p $DATADIR
+chmod 755 $CONSULCONFIGDIR
 chmod 755 $CONSULDIR
-chmod 755 $DATADIR
 chmod 755 $CONFIGDIR
-chmod 755 $CONSULCONFIGDIR
-chkstatus
+chmod 755 $DATADIR
 
-# Check Consul configuration
-curl  ${S3SCRIPT_PATH}/client_json  >  ${CONSULCONFIGDIR}/base.json
+echo "Installing Consul Template..."
+curl -L $CONSUL_TEMPLATE_DOWNLOAD > /tmp/consul_template.zip
+unzip  /tmp/consul_template.zip -d ${BINDIR}/
+chmod 0755 ${BINDIR}/consul-template
+chown root:root ${BINDIR}/consul-template
 chkstatus
 
-echo "Starting Consul with temporary ip -> ($SEEDIP)"
-bash -c "consul agent -config-dir ${CONSULCONFIGDIR} -data-dir ${DATADIR} -retry-join ${SEEDIP} &"
-
-echo "Install Consul Template"
-curl -L $CONSUL_TEMPLATE_DOWNLOAD >  /tmp/consul_template.zip
-unzip  /tmp/consul_template.zip -d  /usr/local/bin
-chmod 0755 /usr/local/bin/consul-template
-chown root:root /usr/local/bin/consul-template
+echo "Installing Consul..."
+curl -L $CONSULDOWNLOAD > /tmp/consul.zip
+unzip  /tmp/consul.zip -d ${BINDIR}/
+chmod 0755 ${BINDIR}/consul
+chown root:root ${BINDIR}/consul
 chkstatus
 
-echo Installing Dnsmasq...
-sudo apt-get -qq -y update
-sudo apt-get -qq -y install dnsmasq-base dnsmasq
+echo "Installing Consul startup scripts..."
+curl $CONSUL_SERVICE_CONF > ${CONSUL_SERVICE_FILE}
+chmod 755 ${CONSUL_SERVICE_FILE}
 
-echo Configuring Dnsmasq...
-sudo sh -c 'echo "server=/consul/127.0.0.1#8600" >> /etc/dnsmasq.d/consul'
-sudo sh -c 'echo "listen-address=127.0.0.1" >> /etc/dnsmasq.d/consul'
-sudo sh -c 'echo "bind-interfaces" >> /etc/dnsmasq.d/consul'
+curl  -s ${S3SCRIPT_PATH}/consul_client_config.json > ${CONSULCONFIGDIR}/client.json.tmp
+sed -i "s/__CONSUL_TAG_KEY__/${CONSUL_TAG_KEY}/" ${CONSULCONFIGDIR}/client.json.tmp
+sed -i "s/__CONSUL_TAG_VALUE__/${CONSUL_TAG_VALUE}/" ${CONSULCONFIGDIR}/client.json.tmp
+mv ${CONSULCONFIGDIR}/client.json.tmp ${CONSULCONFIGDIR}/client.json
 
-echo "Restarting dnsmasq..."
-sudo service dnsmasq restart
+echo "Starting Consul..."
+service consul start
```

Comment:
This command needs to be changed to:
systemctl start consul

@clstokes I tried to update this command but I was unable to commit to your branch